build.sbt

```diff
@@ -8,7 +8,7 @@ sparkVersion := sys.props.getOrElse("spark.version", "1.4.1")
 spName := "graphframes/graphframes"
 
 // Don't forget to set the version
-version := "0.1.0-SNAPSHOT"
+version := "0.1.0-spark" + sparkVersion.value.substring(0, 3)
```

Comment:
If the commit is not for the release, we should change the version back to `-SNAPSHOT`.